### DISCUSSION TITLE
Improve Changelogs regarding Migrations

### DISCUSSION THREAD

**Discussion by Blackskyliner** [jellyfin/jellyfin#11975] [General]
As I just ran into it and it now blocks my ability to use Jellyfin at the moment, I just wanted to discuss if there should maybe an dedicated section to Migrations (especially if blocking the startup) in the Changelog.

I do have some amount of Audio files in my instance and the FixAudio Migration added to the 10.9.4 release now blocks the startup of the instance for an uncertain amount of time.

I wouldn´t even consider to write anything about that here but it just hit me unprepared as this migration was seemingly nowhere mentioned in the Changelog and has an impact on instances which do have some amount of audio files. Or at least I do not read "Mark Audio as RequiresDeserialization and backfill data" as "we will run a blocking migration without any progress indication when jellyfin starts" while looking over the changes of a release. 

Also why do data migrations which iterate over a known number of entries not have any progress indication when they might be finished? This would also be an awesome addition in the future to just put something like that in such migrations like "x of y migrated (z%) - ETA: hh:mm:ss".

**Comment by Shadowghost** [comment]
I totally agree with that this should have been communicated better.
Regarding the progress: #11943 added progress indication and should improve migration speed.
